Description: Use python3 syntax
 Python2 is obsolete.  Update Build.py to use python3 syntax instead.
Author: Steve Langasek <vorlon@debian.org>
Last-Update: 2019-08-30

Index: edk2/UefiCpuPkg/ResetVector/Vtf0/Build.py
===================================================================
--- edk2.orig/UefiCpuPkg/ResetVector/Vtf0/Build.py
+++ edk2/UefiCpuPkg/ResetVector/Vtf0/Build.py
@@ -33,7 +33,7 @@
             'Vtf0.nasmb',
             )
         ret = RunCommand(commandLine)
-        print '\tASM\t' + output
+        print ('\tASM\t' + output)
         if ret != 0: sys.exit(ret)
 
         commandLine = (
@@ -41,7 +41,7 @@
             'Tools/FixupForRawSection.py',
             output,
             )
-        print '\tFIXUP\t' + output
+        print ('\tFIXUP\t' + output)
         ret = RunCommand(commandLine)
         if ret != 0: sys.exit(ret)
 
